### ISSUE TITLE
Let's make this repo to be the easiest way to run a ding-dong-bot.java bot!

### ISSUE THREAD

**Issue by huan** [wechaty/java-wechaty-getting-started#1]
This repository is for any Java developers who want to build a chatbot based on our great Java Wechaty library.

It will be a great help for all Java developers if we can make this repository to be able to ready for use by running:

1. `git clone`
1. `make install`
1. `make bot`

Then everything will be set up.

I have set up the skeleton of this repo, with the `examples/ding-dong-bot.java`, let's finish this repo so that it can be run out-of-the-box! @wechaty/java

**Comment by rdfsource** [comment]
Dear Sirs:
I think we should add tip: Please install MinGW and Git Bash at first except your IDE, when you using Windows System.

**Comment by rdfsource** [comment]
By testing, the Terminal code might be:
git clone https://github.com/wechaty/java-wechaty-getting-started.git
cd .\java-wechaty-getting-started\
i found that npm -install code note work, even make install cannot work.
whether we can add an 'package-lock.json' or not, for us to run the program like wechatty

ths~

**Comment by huan** [comment]
@rdfsource Thank you very much for being interested in our Java Wechaty!

This repo has been just created and it's in **Working In Progress** stage, so please stay tuned by joining our Wechaty Developers' Room on WeChat, and we will let you know when it's ready to use!

P.S. You can also contact @wechaty/java @diaozxin007 if you want to pushing the progress moves faster!

**Comment by diaozxin007** [comment]
@rdfsource  if you interested our java wechaty. U can see https://github.com/diaozxin007/kotlin-wechaty.

**Comment by Seven-Y** [comment]
所以截至目前，java版本的客户端。依然处于开发状态是么？

**Comment by huan** [comment]
Yes! We are an active open source project, and active open source project will always keep developing!

**Comment by Seven-Y** [comment]
那么这一块得V1.0版本大概什么时间可以完善呢？

**Comment by huan** [comment]
We are far away from the v1.0.

In the past 4 years, the Wechaty version comes from v0.0.1 to v0.41.x now.

I hope we could release v1.0 in the following years if we get lucky enough!

**Comment by Seven-Y** [comment]
额。抱歉。可能我的描述有点问题。我的意思是java版本的wechaty。将在什么时候完善呢。目前正在springboot中集成这一块。发现尚有很多东西不完善

**Comment by tmedivh** [comment]
I got a free token, but I cannot make it work. ! 
It reflects an error

>  cannot get ip by token, check token

I read a lot of documents, but now I still don't know what went wrong

**Comment by huan** [comment]
@tmedivh I believe you need to read https://github.com/wechaty/wechaty/issues/1986 and https://github.com/wechaty/wechaty/issues/1985

Good luck!

**Comment by tmedivh** [comment]
Thanks a lot for your info!
